### PR TITLE
Add `HOST` environment variable

### PR DESCRIPTION
**Pull request by teoljungberg**
We're depending on `HOST` to be set in: `app/services/repo_activator.rb`
The default value is the standard url and port for `rails s`.

### CODE REVIEW

File: .sample.env [hunk 1]
```diff
@@ -2,10 +2,11 @@ CHANGED_FILES_THRESHOLD=300
 ENABLE_HTTPS=no
 GITHUB_CLIENT_ID=github_client_id
 GITHUB_CLIENT_SECRET=github_client_secret
+HOST=0.0.0.0:3000
```
Comment (gylaz) [inline .sample.env]: Is this better than just `localhost`?

Comment (teoljungberg) [inline .sample.env]: No idea, I typed it out out of habbit

Comment (salbertson) [inline .sample.env]: `localhost:5000` for Foreman?

**Comment by salbertson** [comment]
@teoljungberg, one comment, looks good. :+1:

**Pull request merged**